sdk/postgresql/Azure.ResourceManager.PostgreSql/src/PostgreSqlFlexibleServers/Generated/Models/DbMigrationStatus.cs

```diff
@@ -0,0 +1,141 @@
+        /// <param name="cdcDeleteCounter"> CDC delete counter. </param>
+        /// <param name="cdcInsertCounter"> CDC insert counter. </param>
+        /// <param name="appliedChanges"> CDC applied changes counter. </param>
+        /// <param name="incomingChanges"> CDC incoming changes counter. </param>
+        /// <param name="latency"> Lag in seconds between source and target during online phase. </param>
+        /// <param name="message"> Error message, if any, for the migration state. </param>
+        /// <param name="serializedAdditionalRawData"> Keeps track of any properties unknown to the library. </param>
+        internal DbMigrationStatus(string databaseName, MigrationDbState? migrationState, string migrationOperation, DateTimeOffset? startedOn, DateTimeOffset? endedOn, int? fullLoadQueuedTables, int? fullLoadErroredTables, int? fullLoadLoadingTables, int? fullLoadCompletedTables, int? cdcUpdateCounter, int? cdcDeleteCounter, int? cdcInsertCounter, int? appliedChanges, int? incomingChanges, int? latency, string message, IDictionary<string, BinaryData> serializedAdditionalRawData)
+        {
+            DatabaseName = databaseName;
+            MigrationState = migrationState;
+            MigrationOperation = migrationOperation;
+            StartedOn = startedOn;
+            EndedOn = endedOn;
+            FullLoadQueuedTables = fullLoadQueuedTables;
+            FullLoadErroredTables = fullLoadErroredTables;
+            FullLoadLoadingTables = fullLoadLoadingTables;
+            FullLoadCompletedTables = fullLoadCompletedTables;
+            CdcUpdateCounter = cdcUpdateCounter;
+            CdcDeleteCounter = cdcDeleteCounter;
+            CdcInsertCounter = cdcInsertCounter;
+            AppliedChanges = appliedChanges;
+            IncomingChanges = incomingChanges;
+            Latency = latency;
+            Message = message;
+            _serializedAdditionalRawData = serializedAdditionalRawData;
+        }
+
+        /// <summary> Name of the database. </summary>
+        [WirePath("databaseName")]
+        public string DatabaseName { get; }
+        /// <summary> Migration db state of an individual database. </summary>
+        [WirePath("migrationState")]
+        public MigrationDbState? MigrationState { get; }
+        /// <summary> Migration operation of an individual database. </summary>
+        [WirePath("migrationOperation")]
+        public string MigrationOperation { get; }
+        /// <summary> Start date-time of a migration state. </summary>
+        [WirePath("startedOn")]
+        public DateTimeOffset? StartedOn { get; }
+        /// <summary> End date-time of a migration state. </summary>
+        [WirePath("endedOn")]
+        public DateTimeOffset? EndedOn { get; }
+        /// <summary> Number of tables queued for the migration of a DB. </summary>
+        [WirePath("fullLoadQueuedTables")]
+        public int? FullLoadQueuedTables { get; }
```

Comment:
For numbers, better to rename to `NumOfFullLoadQueuedTables`, or `CountOfFullLoadQueuedTables`, etc.